app-backend/db/src/main/resources/migrations/V66__add_last_unlock_table.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE last_unlocked (
+    unlocked_time timestamp with time zone not null primary key
```

Comment:
this is a primary key for the free index -- in principle there should only ever be one value in this table, but if we end up with more than one the advisory_lock is looking for the max value, so i want the index to make sure that doesn't become a bottleneck. if we never mess up it costs us the index space for a one row x one column table